README.md

```diff
@@ -38,6 +38,8 @@ I added the Views associated with the Models and Controllers for the Blog App.
 
 I added forms to the Blog app.
 
+I created integration tests for all of the views used in the project. I covered the user stories (or user workflows) that I want my users to experience while taking into consideration the possible errors my users may cause.
+
```

Comment:
## [OPTIONAL]

- [ ] To make your README clean, I think it would be better to remove some README template Descriptions like:

> NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.

> Add at least 2 questions new developers would ask when they decide to use your project.

> Give credit to everyone who inspired your codebase.

> Write a message to encourage readers to support your project

> Describe features you will add to the project

> Mention all of the collaborators of this project.

> Describe how a new developer could make use of your project.

 Kindly remove all of the above texts in your README file

- [ ] Also I this It would be nice to feel all of empty section with your information. Note that only the `FAQ` section is optional

![image](https://user-images.githubusercontent.com/65068771/217391421-1dd914d1-b2d9-4e4c-9d69-1afff374aab4.png)